src/cdk/a11y/focus-trap.ts

```diff
@@ -144,19 +144,24 @@ export class FocusTrap {
    * @param bound The boundary to get (start or end of trapped region).
    * @returns The boundary element.
    */
-  private _getRegionBoundary(bound: 'start' | 'end'): HTMLElement | null {
+  private _getRegionBoundary(bound: 'Start' | 'End'): HTMLElement | null {
```

Comment:
Can we change this back to `'start'` and `'end'`? I'm pretty sure everywhere else we use string literal types we use all lowercase, we can just convert to upper case in the method

Reply:
done :)